src/ocamlorg_frontend/css/styles.css

```diff
@@ -255,3 +255,11 @@ details summary svg svg {
 summary::-webkit-details-marker {
   display: none;
 }
+
+.dropdown {
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+}
+.dropdown-link:hover{
+  background-color:#00308F;
+  border-radius: 5px;
+}
```

Comment:
This should be possible to do in tailwind with `hover:bg-[#00308F]` and choosing the closest available border radius from the tailwind documentation.

Reply:
Alright then. The nearest radius was 6px and it didn't give me what I wanted so I decided to go for 5px with vanilla css. I will fix that